examples/with-emailpassword-vercel/README.md

```diff
@@ -0,0 +1,55 @@
+![SuperTokens banner](https://raw.githubusercontent.com/supertokens/supertokens-logo/master/images/Artboard%20%E2%80%93%2027%402x.png)
+
+# SuperTokens EmailPassword Demo app
+
+This demo app demonstrates the following use cases:
+
+-   Login
+-   Sign up
+-   Logout
+-   Forgot password flow
+-   Session management & Calling APIs
+
```

Comment:
You should add content here explaining that this is specifically for vercel deployment with express app

Reply:
Also add a link to the deployed vercel app?